Mastodon/Scene/Onboarding/Welcome/WelcomeViewModel.swift

```diff
@@ -26,5 +29,40 @@ final class WelcomeViewModel {
             .map { !$0.isEmpty }
             .assign(to: &$needsShowDismissEntry)
     }
-    
+
+    func downloadDefaultServer(completion: (() -> Void)? = nil) {
+            context.apiService.defaultServers()
+            .timeout(.milliseconds(500) , scheduler: DispatchQueue.main)
```

Comment:
❓ What's this timeout for?

Reply:
It's a requirement/design-decision: If getting the list from the server takes longer than 500ms, we fallback to mastodon.social